history-of-the-quakers-v4/original/17-book-7-chapter-11.adoc

```diff
@@ -251,7 +260,7 @@ shall be in any case molested or prejudiced in his or their person or estate,
 because of his or their conscientious persuasion or practice,
 nor be compelled to frequent or maintain any religious worship, place,
 or ministry contrary to his or their minds,
-or to do or to suffer any other act or thing contrary to their religious persuasion.`"
+or to do or to suffer any other act or thing contrary to their religious persuasion.
 
 May it please the Proprietaries,
```

Comment:
Not sure what to do with this in the middle of the embed.

Reply:
Yeah, I think let's just leave it alone 👍